charts/jellyfin/0.1.0/Chart.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: v2
+appVersion: 10.6.4
+description: Jellyfin is a Free Software Media System
+name: jellyfin
+version: 0.1.0
+upstream_version: 4.2.1
+keywords:
+  - jellyfin
+  - plex
+  - emby
+home: https://github.com/truecharts/charts/tree/dev/charts/heimdall
```

```
https://github.com/truecharts/charts/tree/dev/charts/jellyfin
```

---

Yes. thanks :)

Hint: the first button on the edit bar, give you the option to suggest changes which I can merge with one click
Like this:
```suggestion
home: https://github.com/truecharts/charts/tree/dev/charts/jellyfin
```

---

Good tip, I'll use that from now on

---

Awesome, that really saves a lot of time :)